### PR TITLE
Add Super Mario War to Ports

### PR DESCRIPTION
**Pull request by HerbFargus**
script module to install super mario war to ports from the experimental
menu in the RetroPie Setup script.

### CODE REVIEW

File: scriptmodules/ports/Super Mario War.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+rp_module_flags="nobin"
+
+function sources_smw() {
+    gitPullOrClone "$md_build" https://github.com/HerbFargus/smw.git
+}
+
+function build_smw() {
+    cd smw
+    chmod +x configure
+    ./configure
```
Comment (joolswills) [inline scriptmodules/ports/Super Mario War.sh]: You could change the permissions in the repository for configure to avoid this step btw.

**Comment by joolswills** [comment]
cheers.

**Comment by HerbFargus** [comment]
and I was just learning how to squash commits... ha ha I guess you beat me  to it- sorry, I'm still figuring out this whole github thing. Thanks

**Comment by joolswills** [comment]
I missed the merge commits actually. oh well.

You could have just added the chmod commit, and pushed to the branch - wasn't need to squash it, but you could do that also of course, and then force push to update the branch/pull request. I guess you merged from master or so and back again?

**Comment by joolswills** [comment]
I didn't see the updates to the pull request, so please just comment on it so I know not to merge if doing stuff. Cheers.

**Comment by HerbFargus** [comment]
Yeah I'm still getting used to the shell commands, I'll be sure to communicate better what I'm doing in the future :)

**Pull request merged**